Authorization/build.gradle

```diff
@@ -21,9 +22,13 @@ repositories {
 	mavenCentral()
 	maven { url 'https://repo.spring.io/milestone' }
 	maven { url 'https://repo.spring.io/snapshot' }
+	dependencies {
```

Comment:
`dependencies` inside `repositories` looks strange.
Maybe doesn't even do anything.
I suspect it's because the official docs have bad formatting?
![image](https://github.com/NicolayMarunevici/Online-Technical-Store/assets/4677693/10aab9c8-aa2e-4c9c-8774-d1779b6babbd)

But based on the same docs, you don't need this `dependencies` - it's a legacy apporach.
You're using `plugins { id }` syntax from step 2.
Then you need `liquibaseRuntime` below, instead of `implementation`